### ISSUE TITLE
Erweiterung für !text close/open

### ISSUE THREAD

**Issue by samari-k** [FU-Hagen-Discord/fernuni-bot#218]
Für die nächste Klausurenphase fänd ich toll, wenn Boty zusätzlich in den jeweiligen Channeln beim Schließen und Öffnen ne kleine Nachricht hinterlässt. Sowas wie "Dieser Channel ist für die nächsten zwei einhalb Stunden schreibgeschützt. Wir wünschen allen Klausurteilnehmerinnen viel Erfolg!" / "Der Schreibschutz ist ab sofort wieder aufgehoben. Erzählt mal, wie liefs?"

**Comment by Lou-M** [comment]
Das ist mit dem neusten „Klausuren“ Bot geschehen oder? 
Das soll in New Boty integriert werden, @dnns01 ?

**Comment by dnns01** [comment]
Genau. Aber das passiert nachdem alle in Boty bisher bestehenden Feature migriert wurden. Denn der Klausurenbot als Provisorium läuft ja.
